backend/src/main/java/com/htecgroup/skynest/util/UrlUtil.java

```diff
@@ -39,4 +42,23 @@ public class UrlUtil {
               new String[] {USERS_CONTROLLER_URL + LOG_IN_URL})
           .flatMap(Stream::of)
           .toArray(String[]::new);
+
+  public static String SKYNEST_HOST;
+  public static int BACKEND_PORT;
+  public static int FRONTEND_PORT;
+
+  @Value("${skynest.host}")
```

Comment:
you can use ConfigurationProperties annotation and see how can you extract this in separate property class

Reply:
I've moved these properties to a class annotated with ConfigurationProperties.